tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature

```diff
@@ -27,20 +27,20 @@ Feature: restrict Sharing
     And user "Brian" has uploaded file "lorem.txt" to "simple-folder/lorem.txt" in the server
     And user "Brian" has logged in using the webUI
 
-  @smokeTest
+  @smokeTest @skipOnOCIS
   Scenario: Restrict users to only share with users in their groups
+#   shares-only-with-groups-members configuration is not available now in the oCIS webUI
     Given the setting "shareapi_only_share_with_group_members" of app "core" has been set to "yes" in the server
     When the user opens the share dialog for folder "simple-folder" using the webUI
-
     And the user types "Ali" in the share-with-field
     Then "user" "Alice Hansen" should be listed in the autocomplete list on the webUI
     But "user" "Alison Cooper" should not be listed in the autocomplete list on the webUI
 
-  @smokeTest
+  @smokeTest @skipOnOCIS
   Scenario: Restrict users to only share with groups they are member of
+#   shares-only-with-groups-members configuration is not available now in the oCIS webUI
```

Comment:
I didn't mention the issue link in comment because at the top @issue-ocis-1317 tag is present which is related to the issue